website/promotion/emails.py

```diff
@@ -23,3 +24,24 @@ def send_weekly_overview():
             "upcoming_requests": upcoming_requests,
         },
     )
+
+
+def send_daily_overview():
+    for channel in PromotionChannel.objects.filter(
```

Comment:
There is one more thing that would be really nice: grouping the emails by receiver (or otherwise, put the channel name in the subject).

Then it would be roughly
```
for email in PromotionChannel.objects.values_list("publisher_reminder_email", flat=True).distinct():
    daily_promotion = <PromotionRequests for a channel with `publisher_reminder_email=email`>
```